modules/utxo-lib/src/bitgo/psbt/paygoAddressProof.ts

```diff
@@ -0,0 +1,87 @@
+import * as bitcoinMessage from 'bitcoinjs-message';
+import { crypto } from 'bitcoinjs-lib';
+import { ProprietaryKey } from 'bip174/src/lib/proprietaryKeyVal';
+
+import { networks } from '../../networks';
+import { toBase58Check } from '../../address';
+import { getPsbtOutputProprietaryKeyVals, ProprietaryKeySubtype, PSBT_PROPRIETARY_IDENTIFIER } from '../PsbtUtil';
+import { UtxoPsbt } from '../UtxoPsbt';
+
+export const PayGoAddressProofKey: ProprietaryKey = {
+  identifier: PSBT_PROPRIETARY_IDENTIFIER,
+  subtype: ProprietaryKeySubtype.PAYGO_ADDRESS_PROOF,
+  // Is there a better way to not provide a pubkey?
+  keydata: Buffer.from([]),
```

Comment:
```suggestion
  // Is there a better way to not provide a pubkey?
  keydata: Buffer.from([]),
```
maybe what is encoded is the pubkey used for the verification